src/styles/blocks/container.scss

```diff
@@ -0,0 +1,46 @@
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 1fr);
+
+  @media (min-width: 600px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (min-width: 900px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
```

Comment:
sorry, better to make this task only with 1 media query. With so many media queries there is no sense to use grid